megamek/src/megamek/common/preference/ClientPreferences.java

```diff
@@ -108,6 +116,7 @@ public ClientPreferences(IPreferenceStore store) {
         store.setDefault(START_SEARCHLIGHTS_ON, true);
         store.setDefault(USER_DIR, "");
         store.setDefault(MML_PATH, "");
+        store.setDefault(MML_PATH, "");
```

Comment:
Just checking if this is a type - as it's a duplicate statement... should this be a different path here?  Should it be MML_PATH?